### ISSUE TITLE
interleav_wrap has no padding bug

### ISSUE THREAD

**Issue by liuheng0111** [InternLM/InternLM-XComposer#202]
![image](https://github.com/InternLM/InternLM-XComposer/assets/18352727/6aa60541-3aa9-4d42-83af-67db53c9e2ab)
There is no padding when wrap_embeds = torch.cat(wrap_embeds_list) in interleav_wrap function, Is this a bug in it?

**Comment by TClan8023** [comment]
Hi，have you solved this question? I met the same question.

**Comment by yuhangzang** [comment]
The default batch size is 1 thus no padding.

**Comment by Starfulllll** [comment]
> The default batch size is 1 thus no padding.

Hi, if I want to train or inference with a larger batch size, how do I modify the code?

**Comment by decreasbetter** [comment]
> > 默认批量大小为 1，因此没有填充。
> 
> 你好，如果我想用更大的批量进行训练或推理，我该如何修改代码？

你好，请问你现在解决padding的问题了吗？padding的数值应该是多少呢？
